index.html

```diff
@@ -4,18 +4,218 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <!-- Add a link to your custom CSS here -->
-
-    <title>Responsive grid project</title>
+    <link rel="stylesheet" href="../css/main.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css" integrity="sha512-Fo3rlrZj/k7ujTnHg4CGR2D7kSs0v4LLanw2qksYuRlEzO+tcaEPQogQ0KaoGN26/zrn20ImR1DfuLWnOo7aBA==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Estonia&display=swap" rel="stylesheet">
+    <title>Plants</title>
 </head>
 
 <body>
+    <div class="container">
+        <header class="pd">
+        
+            <div class="search-box">
+                <div>
+                    <i class="fas fa-search search-icon "></i>
+                </div>
+                <div>
+                    <input class="search-bar" type="text" placeholder="Search...">
+                </div>
+                
+                
+            </div>
+                
+            <div>
+                
+                <i class="fas fa-shopping-bag"></i>
+                
+            </div>
+           
+            
+        </header>    
+            
+            
+        <nav>
+            <div class="intro">
+                <h1 class="red-brown-text">PRICKLES & CO.</h1>
+                <p>We Deliver To Your Doorstep</p>
+                <p>Call us on 020-55-991-07</p>
+            </div>
+            <ul class="links">
+                <li><a href=""></a>Shop</li>
+                <li><a href=""></a>About</li>
+                <li><a href=""></a>FAQ</li>
+                <li><a href=""></a>Contact</li>
+            </ul>
+            <div class="hamburger">
+                <div class="line"></div>
+                <div class="line"></div>
+                <div class="line"></div>
+            </div>
+        </nav>
+        
+        <section class="main">
+            <div>
+                <div class="text_2"><span class="logo-text1">ESTB</span><img src="../images/image.svg" alt="logo-image" class="logo-image" ><span class="logo-text2">2020</span></div>
```

Comment:
This section with the two logos texts and the image is done using position: absolute, how could we do this with flex?